src/core/BaseEfTask.py

```
@@ -26,7 +26,9 @@
 from src.config import config as app_config
 
 # 覆写框架截图时间戳格式：日期_时分秒（无毫秒）
-import ok.gui.debug.Screenshot as _ok_screenshot
+# 注意：ok-script 新版中 ok.gui.debug.Screenshot 只是 ok.core.screenshot 的
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)